src/assets/toolpanel/message-start-event.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="22" height="22" viewBox="0 0 22 22" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
end-email-event.svg looks like this:
<img width="150" alt="Screen Shot 2019-10-23 at 11 14 52 AM" src="https://user-images.githubusercontent.com/6653340/67408069-73478300-f586-11e9-83cc-144770060d57.png">


Should the envelops in the toolpanel be updated to look all the same?

Reply:
Good point; I can update it to keep this consistent.